<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="X390" ignored="0" name="Test" tests="4" failures="3" timestamp="2022-07-19T22:25:19 EDT" time="0.047" errors="0">
  <testcase name="TestNG3" time="0.007" classname="com.runFaileTestCase1.demo2">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.runFaileTestCase1.demo2.TestNG3(demo2.java:10)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TestNG3 -->
  <testcase name="TestNG4" time="0.001" classname="com.runFaileTestCase1.demo2">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.runFaileTestCase1.demo2.TestNG4(demo2.java:15)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TestNG4 -->
  <testcase name="TestNG1" time="0.001" classname="com.runFaileTestCase1.demo1"/>
  <testcase name="TestNG2" time="0.001" classname="com.runFaileTestCase1.demo1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.runFaileTestCase1.demo1.TestNG2(demo1.java:15)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TestNG2 -->
</testsuite> <!-- Test -->
